assets/js/components/UI/CodedTerm/Item.jsx

```diff
@@ -2,6 +2,8 @@ import React from "react";
 import PropTypes from "prop-types";
 
 const UICodedTermItem = ({ item = {} }) => {
+  if (!item) return null;
```

Comment:
Not that it matters, but this condition will never return `true` with the `item = {}` default declaration above.  Probably could remove it from line 4 to read `({ item })` instead.

Reply:
Hmm. We added it because we we're getting an error.

Reply:
Weird.  An empty object literal shouldn't return `undefined` or `null` or be falsy.

Reply:
...unless maybe the item being passed in is `null`, then maybe it'd bypass the default value?  Probably not that big a deal.